common.mk

```diff
@@ -45,7 +51,8 @@ $(FIRRTL_FILE) $(ANNO_FILE): generator_temp
 	@echo "" > /dev/null
 
 # AG: must re-elaborate if ariane sources have changed... otherwise just run firrtl compile
-generator_temp: $(SCALA_SOURCES) $(ARIANE_VLOG_SOURCES) $(sim_files)
+generator_temp: $(SCALA_SOURCES) $(PROJECT_GENERATOR_SOURCES) $(sim_files)
```

Comment:
I think the `PROJECT_X` naming scheme for the additional flags can easily be confused with the `PROJECT` and `SUBPROJECT` make variables, which are unrelated.

What about `MISC_GENERATOR_SOURCES` or `EXTRA_GENERATOR_SOURCES`

Reply:
Agreed. I think we can do `EXTRA`.